megamek/src/megamek/common/Terrain.java

```diff
@@ -386,29 +388,52 @@ public int movementCost(Entity e) {
                 } else {
                     mp = 1;
                 }
+
+                if (isCrossCountry) {
+                    if (((moveMode == EntityMovementMode.HOVER) || (moveMode == EntityMovementMode.TRACKED))
+                            && (level == 6)) {
+                        mp *= 2;
+                    } else if ((moveMode == EntityMovementMode.WHEELED)) {
+                        mp *= 2;
+                    }
+                }
+
                 if ((e instanceof Mech) && e.isSuperHeavy()) {
                     mp -= 1;
                 }
+
                 if (e.hasAbility(OptionsConstants.PILOT_TM_MOUNTAINEER)) {
                     mp -= 1;
                 }
+
                 if ((e.hasAbility(OptionsConstants.INFANTRY_FOOT_CAV)
                         && (moveMode == EntityMovementMode.INF_LEG))) {
                     mp -= 1;
                 }
                 return Math.max(0, mp);
             case Terrains.WOODS:
                 mp = level;
+                if (isCrossCountry) {
+                    if ((level == 1) && ((moveMode == EntityMovementMode.HOVER) || (moveMode == EntityMovementMode.WHEELED))) {
+                        mp *= 2;
+                    } else {
+                        mp *= 2;
```

Comment:
This just assigns the same value for both.

Reply:
Tracked vehicles are allowed in light woods while the others aren't

Reply:
That isn't what this code does. It just assigns mp to mp * 2 if isCrossCountry is true.

Reply:
Fixed